test/cppsuite/tests/test_template.cpp

```diff
@@ -28,60 +28,84 @@
 
 #include "test_harness/test.h"
 
+/* Class that specifies what to write into the tracking table. */
+class tracking_table_template : public test_harness::workload_tracking {
+
+    using workload_tracking::workload_tracking;
```

Comment:
Can we avoid using this? It's not something we usually do

Reply:
Fixed in 552d16167efbf8ed9d5ff03e0e6853b6eb8f44e9